package.json

```diff
@@ -9,7 +9,10 @@
     "db:migrate": "drizzle-kit migrate",
     "db:push": "drizzle-kit push",
     "db:studio": "drizzle-kit studio",
-    "db:seed": "dotenv npx tsx ./src/server/db/seed.ts",
+    "db:seed:educations": "dotenv npx tsx ./src/server/db/seed/educations.seed.ts",
+    "db:seed:license": "dotenv npx tsx ./src/server/db/seed/license.seed.ts",
+    "db:seed:users": "dotenv npx tsx ./src/server/db/seed/user.seed.ts",
+    "db:seed:all": "dotenv npx tsx ./src/server/db/seed/index.ts",
```

Comment:
Did you maybe test all use-cases? When using `db:seed:all` is it trying to run seed scripts twice or just once?

Reply:
I've tested them now. The only problem is that when I run them individually they won't stop executing in command line even though they have seeded the tables.

Reply:
Than end `then` block with 
```typescript
console.log("Done seeding ... file"); 
process.exit(0)
```

Reply:
When I add that and try to run `index.ts` somehow that `then` is triggered for the first function that is called inside it.
For example `await getLicenses();` in index.ts seems to call
```ts
getLicenses()
  .then((licenses) => {
    console.log("Done seeding lic file");
    process.exit(0);
  })
  .catch((err) => {
    console.log(err);
    process.exit(1);
  });
```
at the end of the `license.seed.ts` instead of the `getLicenses` function directly

Reply:
That is what I thought. You can maybe google for another solution, or try to ask chatGPT. I will also investigate a little to try and find out the best approach

Reply:
It seems to be working now.